README.md

```diff
@@ -31,8 +31,21 @@ Create a new site
 gatsby new <PROJECT_NAME> https://github.com/oberonamsterdam/gatsby-starter-oberon
 ```
 
+Husky
+
+Because of [this](https://github.com/gatsbyjs/gatsby/issues/19589) we need to install husky manually after running `gatsby new`
```

Comment:
Ook nog even in de setup guide zetten
https://oberon.papyrs.com/Front-end-project-setup-guide

Reply:
döner 👍🏻